### DISCUSSION TITLE
AI 的五大預測！人工智能未來的發展會是怎樣？

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#987] [General]
既我之前所講到的問題，雖然我對AI在進化的議題裡面是比較在乎社會議題那一方面的，但是好奇心也是會讓我想要知道AI會來會長怎麼樣子，於是我就上網查到了一篇文章裏面是分享AI未來的的發展，我自己是認為文章內容篇理想，因為在這些AI發展的過程中，就會受到社會學家的譴責，所以我覺得要在短時間內有突破性的進化是不太可能的
歡迎各位分享自己的想法~~
https://www.innopreneur.io/blogs/innovative-product-technology/ai-predictions-2021/

**Comment by ruthchen1020** [comment]
我是採取比較悲觀的想法，因為美國跟中國軍方根本無視法律的存在，更不可能理甚麼社會學家的譴責。軍方的科技都是超乎民間很多的。

**Comment by manipulateed** [comment]
AI的發展趨勢看似都還在人類可以掌控的層面下去運用，所以看起來相對樂觀，撇除倫理道德的譴責，AI確實造福許多商業。

**Comment by Sylvie0601** [comment]
我覺得AI還是要在人類的掌控才能繼續運用，若是哪一天他們逃離人類的掌控的話，不知道會發生甚麼事情，既然都逃脫人類掌控的話，那是否會對人類造成甚麼威脅呢?

**Comment by Tuoresa** [comment]
一直都覺得AI是一件讓人不由讚嘆科技發展的卓越與無可限量性，
卻又非常模糊人類與科技之界線的領域。

文章內提到的未來發展若根據現狀的快速發展感覺不無可能，
但不知道人類最終想利用AI做到什麼程度，
永無止盡的上綱感覺會造成無法收拾的結果

人類竭盡全力發展的成果最終會取代甚至吞噬人類，
好像是一件很神奇的是哈哈
